examples/sidekiq.yml

```diff
@@ -1,5 +1,18 @@
 # sidekiq process
 
+# NOTE: this config assumes that `config/sidekiq.yml` declares
```

Comment:
I feel this is outside the scope of this file. Instead how about adding a comment on the same line as the eye config that says `# must correspond to pidfile specified in sidekiq's config`